### PR TITLE
feat: Added Homepage

### PR DESCRIPTION
**Pull request by parthgabani777**
Added homepage

### CODE REVIEW

**Review by ajay-mandaviya — APPROVED** [review]
Everything is well structure

File: css/home.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+.category-card{
+      min-width: 27rem;
+      min-height: 23rem;
+      width: fit-content;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      background: no-repeat center center;
+      background-size: cover;
+      box-shadow: inset 0 0 200px #000;
```
Comment (amarphule) [inline css/home.css]: you can use color variable

**Review by amarphule — CHANGES_REQUESTED** [review]
GIven suggestion, please check

File: css/home.css (hunk 1 above)
Comment (parthgabani777) [reply css/home.css]: thanks for the suggestion I forgot to use a color variable.

**Review by Kavita539 — APPROVED** [review]
Looks good to merge!

**Review by spraveenofficial — APPROVED** [review]

**Pull request merged**